### ISSUE TITLE
[CRASH] The server crash occurs on boot on Arclight server with Lithium mod installed.

### ISSUE THREAD

**Issue by cptDeeJay** [jellysquid3/Lithium#575]
**Lithium Version :**
lithium-neoforge-0.14.0-snapshot+mc1.21.1-build.88.jar

**Arclight version :** 
诸王 (Feudal Kings) / arclight-1.21.1-1.0.0-SNAPSHOT-78c1bfe

**OS & Java versions :**
CasaOS [Crafty] (Debian) java version 21.0.3

**Summary :**
It has been observed that the Arclight 1.21.1 with commit ([78c1bfe](https://github.com/IzzelAliz/Arclight/commit/78c1bfef53d017c7d67576d6649d52a13a303668)) crashes on boot with Lithium mod installed (NeoForge). It has been confirmed that the issue is not occuring on standalone NeoForge (Version 21.1.73) and with [Lithium](https://www.curseforge.com/minecraft/mc-mods/lithium/files/5817196) mod disabled. Please note that the same ticket has been made on Arclight bug tracker with Id `[#1535](https://github.com/IzzelAliz/Arclight/issues/1535)`

**Step to reproduce :**
1. Install Arclight 1.21.1 with commit ([78c1bfe](https://github.com/IzzelAliz/Arclight/commit/78c1bfef53d017c7d67576d6649d52a13a303668)
2. Install Lithium mod version 0.14.0
3. Run the server
4. Observe the issue.

**Comment by 2No2Name** [comment]
Possible workaround:
Add the line `mixin.ai.poi.fast_portals=false` to the lithium.properties file in the config folder.
Please report back if this works around the issue.

**Comment by cptDeeJay** [comment]
Yes, with the dvar **`mixin.ai.poi.fast_portals=false`** added to the lithium.properties, the issue no longer occurs on linked versions of Lithium mod and Arclight server. 
Thank you very much.

**Comment by cptDeeJay** [comment]
@2No2Name Should I close the ticket as workaround is working or there is any further work planned on this issue ?

**Comment by 2No2Name** [comment]
I hope the author of arclight server will disable the setting for all users as described here:
https://github.com/CaffeineMC/lithium-fabric/wiki/Disabling-Lithium's-Mixins-using-your-mod's-fabric.mod.json-or-neoforge.mods.toml

Maybe there is a way to get proper compatibility without disabling the setting. But I haven't looked at what arclight server actually does
